src/fuzz_introspector/datatypes/fuzzer_profile.py

```diff
@@ -454,9 +454,6 @@ def _load_coverage(self, target_folder: str) -> None:
                 self.coverage.correlate_jvm_method_with_coverage(
                     self.all_class_functions
                 )
-
```

Comment:
The changes in this file reverse what we just pushed here: https://github.com/ossf/fuzz-introspector/pull/634

I assume this should not be the case?

Reply:
Yes, you are right, I also just found that out in my testing and verification and then I see your message.